### PR TITLE
Modify S6373: Change text to the LaYC format (APPSEC-1108)

### PR DESCRIPTION
**Pull request by gaetan-ferry-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

File: rules/S6373/java/how-to-fix/java-se.adoc [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+=== Code examples
+
+include::../common/code-rationale.adoc[]
+
+==== Noncompliant code example
+
+[source,java,diff-id=1,diff-type=noncompliant]
+----
+
```
Comment (sebastien-andrivet-sonarsource) [inline rules/S6373/java/how-to-fix/java-se.adoc:11]: `import javax.xml.parsers.SAXParserFactory;` is missing here or should be removed in the compliant code.

File: rules/S6373/java/rule.adoc [hunk 2]
```diff
@@ -1,112 +1,78 @@
...
 
-== Resources
+=== Standards
 
-* https://docs.oracle.com/en/java/javase/13/security/java-api-xml-processing-jaxp-security-guide.html#GUID-8CD65EF5-D113-4D5C-A564-B875C8625FAC[Oracle Java Documentation] - XML External Entity Injection Attack
-* https://owasp.org/www-project-top-ten/2017/A4_2017-XML_External_Entities_(XXE)[OWASP Top 10 2017 Category A4] - XML External Entities (XXE)
-* https://cheatsheetseries.owasp.org/cheatsheets/XML_External_Entity_Prevention_Cheat_Sheet.html#java[OWASP XXE Prevention Cheat Sheet]
-* https://cwe.mitre.org/data/definitions/611[MITRE, CWE-611] - Information Exposure Through XML External Entity Reference
-* https://cwe.mitre.org/data/definitions/827[MITRE, CWE-827] - Improper Control of Document Type Definition
+* OWASP - https://owasp.org/www-project-top-ten/2017/A4_2017-XML_External_Entities_(XXE)[OWASP Top 10 2017 - Category A4 - XML External Entities (XXE)]
```
Comment (sebastien-andrivet-sonarsource) [inline rules/S6373/java/rule.adoc:73]: The OWASP Top 10 2021 is missing (probably A05:2021)

File: rules/S6373/java/rule.adoc [hunk 3]
```diff
@@ -1,112 +1,78 @@
...
-----
-XMLInputFactory factory = XMLInputFactory.newInstance();
-factory.setProperty("http://apache.org/xml/features/xinclude", true);
-factory.setXMLResolver(new MySafeEntityResolver());
-----
+* OWASP - https://cheatsheetseries.owasp.org/cheatsheets/XML_External_Entity_Prevention_Cheat_Sheet.html#java[OWASP XXE Prevention Cheat Sheet]
+* Java documentation - https://docs.oracle.com/en/java/javase/13/security/java-api-xml-processing-jaxp-security-guide.html#GUID-8CD65EF5-D113-4D5C-A564-B875C8625FAC[XML External Entity Injection Attack]
 
-== Resources
+=== Standards
```
Comment (sebastien-andrivet-sonarsource) [inline rules/S6373/java/rule.adoc:71]: Strictly speaking, XInclude is not part of the XML standard but a separate one. So I would add the XInclude standard here: https://www.w3.org/TR/xinclude-11/

**Review by sebastien-andrivet-sonarsource — APPROVED** [review]
This PR is missing in the APPSEC ticket. Except for some details, LGTM.

File: rules/S6373/java/rule.adoc (hunk 3 above)
Comment (gaetan-ferry-sonarsource) [reply rules/S6373/java/rule.adoc:71]: I'd rather keep the standard section for security standard things. I'll put this one in the documentation section.

**Pull request merged**